starter-code/public/admin.html

```diff
@@ -11,7 +11,18 @@
     <link rel="stylesheet" href="/styles/modules.css">
     <link rel="icon" type="image/x-icon" href="https://avatars2.githubusercontent.com/u/18607604?v=3&s=200">
     <!-- TODO: Setup a template for each row of the Author Stats section of this page: -->
-
+    <script id="blog-stats" type="text/x-handlebars-template">
```

Comment:
you can't use the `id="blog-stats"` because it's being used down below, could use `id="blog-stats-template"`

too much stuff in the template, all you needed was a template expression for the `author` and `numWords` object properties that the `Article.numWordsByAuthor`function returns
```
<li>
  <ul>
    <li>{{author}}</li>
    <li>{{numWords}}</li>
  </ul>
</li>
```